llvm/test/Transforms/SimplifyCFG/X86/hoist-loads-stores-with-cf.ll

```diff
@@ -728,6 +726,34 @@ if.true:
   ret i32 %res
 }
 
+define void @diamondCFG(i32 %a, ptr %c, ptr %d) {
```

Comment:
Probably remove this? This function tests the same thing as `single_predecessor`

Reply:
Done.